### PR TITLE
Update voting documentation

### PR DESCRIPTION
**Pull request by dehodson**
Update the readme to include the emojis introduced by #61 and #136

### CODE REVIEW

**Comment by hongaar** [comment]
\*looking for timstamps\*

File: README.md [hunk 1]
```diff
@@ -55,8 +55,8 @@ Votes on a PR are determined through following mechanism:
   their comments, effectively changing their vote.
   
 Emoji which count as votes are:
-* +1: :+1: :heart: :tada:
-* -1: :-1: :confused:
+* +1: :+1: :+1: :thumbsup: :heart: :two_hearts: :blue_heart: :purple_heart: :green_heart: :yellow_heart: :heartpulse: :sparkling_heart: :tada: :heart_eyes: :grin: :grinning: :joy: :ok_hand: :fire: :metal: :raised_hands: :100: :heavy_check_mark: :white_check_mark: :ballot_box_with_check: :accept: :congratulations:
```
Comment (phil-r) [inline README.md]: `:+1:` is duplicated

**Comment by dehodson** [comment]
@hongaar no bamboozles!
@phil-r fixed.

**Comment by hongaar** [comment]
bamboozle free guarantees not possible if chaos

**Pull request merged**